notebooks/en/enterprise_cookbook_argilla.ipynb

```diff
@@ -0,0 +1,1404 @@
+{
```

Comment:
<p>It would be clearer to align the title with the one in the toctree ("Data annotation with Argilla Spaces") to avoid any confusion when referencing the recipe.</p><p> </p><p>Currently it doesn't show up as a list because of how it is formatted.</p><p> </p><p>"In this notebook, we:</p><p>1.</p><p>2.</p><p>3.</p><p>4.</p><p>5."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>